xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -55,7 +55,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Represents the set of captures made by a single capturing group.</summary>
+    <summary>Represents the set of captures made by a single capturing group.  The collection is immutable (read-only) and has no public constructor. The <see cref="T:System.Text.RegularExpressions.CaptureCollection" /> object contains one or more <see cref="T:System.Text.RegularExpressions.Capture" /> objects.</summary>
```

Comment:
This information does not sound relevant for the summary. Also, is it accurate?: What if the regex does not have any matches? Will it still have at least one object?:

`The <see cref="T:System.Text.RegularExpressions.CaptureCollection" /> object contains one or more <see cref="T:System.Text.RegularExpressions.Capture" /> objects.`

Reply:
Update:
The second and third sentence in the summary area _already_ in the remarks.
I suggest the second sentence remains in the summary, the third one should be in the remarks.